src/CrandallGridWorld/TeamBug.java

```diff
@@ -1,62 +1,59 @@
-package CrandallGridWorld;
+package ourTeam;
 
+import java.lang.reflect.Constructor;
 import java.util.ArrayList;
 
 import info.gridworld.actor.Actor;
 import info.gridworld.actor.Critter;
 import info.gridworld.grid.Location;
 
-//Version 1.1.0
+//Version 1.11
 //@Author Suchir/Billy/Tim, and everyone who helped figure this out on the whiteboard
 //Any edits/improvements are appreciated
-//Not sure if we need final for the methods, but it seemed useful.
-
+//Not sure if we need final for the methods, but it seemed useful
 public abstract class TeamBug extends Critter {
 	private int team;
-	private boolean isSterile;
 	private int turnsToDeath;
+	private int turnsToUnfreeze;
+	private boolean isSterile;
 	private boolean isInfected;
 	private boolean canKill;
-	private int turnsToUnfreeze;
 
 	public static final int GEEK = 1;
 	public static final int NERD = 2;
 
-	// Remember, your constuctors need to have a class-type Integer,Boolean
-	// Constuctor
+	public TeamBug() {

```

Comment:
I'm pretty sure we agreed that this was not needed. the constuctor should have a team and hasBred every time....

Are you not using the latest codebase?